src/main/java/uk/gov/gchq/palisade/util/ResourceBuilder.java

```diff
@@ -99,15 +97,20 @@ public static LeafResource create(final URI uri, final ConnectionDetail connecti
     // Throw IllegalArgumentException if unsupported scheme
     public static Resource create(final URI uri, final Map<String, String> attributes) {
         // If passed relative paths, we can resolve them against the user.dir system property
-        URI absolute = uri.isAbsolute() ? uri : ROOT.resolve(uri);

```

Comment:
similarly here seems more readable than an if/else block

Reply:
I would still say the more verbose if statements are easier to read. Especially given anyone who can code understands if statements. Not everyone understands the ternary notation